studio/components/ui/CodeEditor/CodeEditor.tsx

```diff
@@ -60,7 +60,7 @@ const CodeEditor: FC<Props> = ({
     await timeout(500)
     editor?.focus()
     editorRef.current = editor
-  }
+  }, [editorRef?.current, onInputRun])
```

Comment:
My understanding is that ref's shouldn't be used in dependencies since they won't trigger a re-render. Would you mind double checking/confirming this?

Reply:
> My understanding is that ref's shouldn't be used in dependencies since they won't trigger a re-render. Would you mind double checking/confirming this?

Oops, sorry for the mistake! You are correct about passing the dependency in a React hook. 🙏

Ref: [Synchronizing with Effects](https://react.dev/learn/synchronizing-with-effects)

By the way, I have removed the unused code for the `stopPropagation` function.
Let me know if there's anything else I need to change 😄